controller_interface/include/controller_interface/controller_interface.hpp

```diff
@@ -93,13 +93,41 @@ class ControllerInterface : public rclcpp_lifecycle::node_interfaces::LifecycleN
   update() = 0;
 
   CONTROLLER_INTERFACE_PUBLIC
-  std::shared_ptr<rclcpp_lifecycle::LifecycleNode>
-  get_lifecycle_node();
+  std::shared_ptr<rclcpp::Node>
+  get_node();
+
+  /**
+   * The methods below are a substitute to the LifecycleNode methods with the same name.
+   * We cannot use a LifecycleNode because it would expose the possiblity of changing the state
+   * the rest of the ROS system by exposing the change state service.
+   * Changing state of a controller means having been assigned interfaces by the ResourceManager,
+   * and only the ControllerManager should have the possibility of doing it.
```

Comment:
```suggestion
   * Only the Controller Manager should have possibility to change state of s controller.
```

Trying to simplify this...